python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -529,15 +538,17 @@ def __init__(cls, name, bases, this_dict):
 
         # all Ganga classes must have (even empty) schema
         if cls._schema is None:
-            cls._schema = Schema.Schema(None, None)
+            cls._schema = Schema(None, None)
 
         this_schema = cls._schema
 
+        cls.__slots__ = ('__dict__', '_proxyObject')
```

Comment:
We want to move away from coupling the proxy classes and non-proxy classes in `Objects.py`.